CONTRIBUTING.md

```diff
@@ -1,31 +1,62 @@
 # Code Contribution Guidelines
 
-Please see the [Mattermost Contribution Guide](http://docs.mattermost.com/developer/contribution-guide.html) which describes the process for making code contributions across Mattermost projects. 
+Thank you for your interest in contributing! Please see the [Mattermost Contribution Guide](http://docs.mattermost.com/developer/contribution-guide.html) which describes the process for making code contributions across Mattermost projects. 
 
 ### Review Process for this Repo
 
-After following the steps in the [Contribution Guide](http://docs.mattermost.com/developer/contribution-guide.html), submitted pull requests go through the review process outlined below. We aim to start reviewing pull requests in this repo the week they are submitted, but the length of time to complete the process will vary depending on the pull request.
+When you submit a pull request, it goes through the review process outlined below. We aim to start reviewing pull requests in this repo the week they are submitted, but the length of time to complete the process will vary depending on the pull request.
 
 The one exception may be around release time, where the review process may take longer as the team focuses on our [release process](https://docs.mattermost.com/process/release-process.html). 
 
-#### `Stage 1: PM Review`
+#### PR submitted
+
+After a PR is submitted, a core committer applies labels and notifies PMs that there is a PR awaiting review by posting in the [PM/Docs PR Review channel](https://pre-release.mattermost.com/core/channels/pmdocs-pr-review-pub), which is a channel to discuss community pull requests that need review by PMs.
+
+Then, one or more of the labels is applied:
+ - `Awaiting PR`: Applied if the PR is awaiting another to be merged. For example, when a client PR is awaiting a server PR to be merged first. Once the PR is no longer blocked, the core committer removes the `Awaiting PR` label
+ - `1: PM Review`: Applied if the PR has UI changes or functionality that PMs can test on test servers called "spinmints"
+ - `Setup Test Server`: Applied if the PR is queued for PM testing
+ - `Work in Progress`: Applied if the PR is unfinished and needs further work before it's ready for review
+
+#### Stage 1: PM Review
 
 A Product Manager will review the pull request to make sure it:
```

Comment:
Product Manager shouldn't really be capitalized, it's not a proper noun (I know this wasn't changed in the PR but I noticed it :P)